packages/ckeditor5-emoji/tests/manual/emoji.js

```diff
@@ -6,17 +6,27 @@
 /* globals console:false, window, document */
 
 import ClassicEditor from '@ckeditor/ckeditor5-editor-classic/src/classiceditor.js';
+import { Emoji } from '../../src/index.js';
+
+import Clipboard from '@ckeditor/ckeditor5-clipboard/src/clipboard.js';
 import Enter from '@ckeditor/ckeditor5-enter/src/enter.js';
-import Typing from '@ckeditor/ckeditor5-typing/src/typing.js';
+import Mention from '@ckeditor/ckeditor5-mention/src/mention.js';
 import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph.js';
+import Typing from '@ckeditor/ckeditor5-typing/src/typing.js';
 import Undo from '@ckeditor/ckeditor5-undo/src/undo.js';
-import Clipboard from '@ckeditor/ckeditor5-clipboard/src/clipboard.js';
-import Emoji from '../../src/emoji.js';
 
 ClassicEditor
 	.create( document.querySelector( '#editor' ), {
-		plugins: [ Typing, Paragraph, Undo, Enter, Emoji, Clipboard ],
-		toolbar: [ 'undo', 'redo' ]
+		plugins: [
+			Emoji,
+			Clipboard,
+			Enter,
+			Mention,
+			Paragraph,
+			Typing,
+			Undo
+		],
```

Comment:
```suggestion
		plugins: [
			Emoji,
			Mention,
			Paragraph
		],
```

We can reduce an array of plugins by loading the `Essentials` plugin.

https://github.com/ckeditor/ckeditor5/blob/db95303a82cb0aa649bc2d16920b71f531c1fc66/packages/ckeditor5-essentials/src/essentials.ts#L41

This would reduce entries in pkg.json too.